test/calendar_test.js

```diff
@@ -153,10 +153,10 @@ describe("Calendar", () => {
       state = {
         weeks: Calendar.monthWeekTable(2017, 1),
         holidays: [{
-          start: "2017-01-02T00:00:00.000Z",
+          start: "2016-12-26T00:00:00.000Z",
```

Comment:
Probably because it is already tested here: https://github.com/julienw/forms-extension/pull/15/files#diff-4a52023af2254c0d28ddc8e88036ff61R245